wagtail/admin/ui/tables/pages.py

```diff
@@ -90,6 +91,7 @@ def __init__(
         use_row_ordering_attributes=False,
         parent_page=None,
         show_locale_labels=False,
+        actions_next_url=None,
```

Comment:
Reminds me of both "props drilling" and `useContext` in React with the way we pass `actions_next_url` all the way down to the `Cell`'s context data here, but there's not much we can do about it at the moment 😄